package gazebo.swarm.msgs;

/// \ingroup swarm_msgs
/// \interface Datagram
/// \brief A message containing some payload to communicate among one or
/// multiple teammates.

message Datagram
{
  required string src_address = 1;
  required string dst_address = 2;
  required uint32 dst_port    = 3;
  required string data        = 4;
  repeated string neighbors   = 5;
}
